cargo-dist/src/platform.rs

```diff
@@ -829,33 +832,49 @@ fn max_of_min_libc_versions(
 /// Compute the requirements for running the binaries of this release on its host platform
 fn native_runtime_conditions_for_artifact(
     dist: &DistGraphBuilder,
-    artifact_id: &ArtifactId,
+    artifact: &Artifact,
 ) -> RuntimeConditions {
     let manifest = &dist.manifest;
     let mut runtime_conditions = RuntimeConditions::default();
+    let artifact_id = &artifact.id;
+
     if let Some(artifact) = manifest.artifacts.get(artifact_id) {
         for asset in &artifact.assets {
             let asset_conditions = native_runtime_conditions_for_asset(manifest, &asset.id);
             runtime_conditions.merge(&asset_conditions);
         }
     };
+
     // FIXME: in our test suite we're running bare artifacts=global so we're missing
     // all artifact/linkage info, preventing basic glibc bounds
```

Comment:
is this true? if so, does this change affect this comment?

Reply:
Yes, it looks like we are in fact using artifacts=global in some situations.

As I suspected, this change (and technically the PR adding minimum-glibc-version) means the comment should be moved.